frag-nav/src/main/java/com/ncapdevi/fragnav/FragNavController.kt

```diff
@@ -663,10 +674,13 @@ class FragNavController constructor(private val fragmentManger: FragmentManager,
                 setTransition(options.transition)
 
                 options.sharedElements.forEach { sharedElement ->
-                    addSharedElement(
-                            sharedElement.first,
-                            sharedElement.second
-                    )
+                    // TODO: Better null handling
+                    if (sharedElement.first != null) {
+                        addSharedElement(
+                                sharedElement.first!!,
```

Comment:
`addSharedElement` now requires a non-null view.

Reply:
The safest way to do sharedElement.first!!  is either calling firstOrNull() like  sharedElement?.firstOrNull() or to getOrNull(0)